explorer-ui/src/components/Activities/Activities.jsx

```diff
@@ -0,0 +1,146 @@
+// This function calls API methods by fetch function (you can use XMLHttpRequest or $.ajax instead):
+import Grid from "../Grid/Grid";
+import { useState, useEffect } from "react";
+import Card from "../Card/Card";
+import "./Activities.css"
+import AttractionContainer from "./AttractionContainer";
+//You should get your API key at https://opentripmap.io
+const apiKey = "5ae2e3f221c38a28845f05b66afc7a4b942f1b2a702f9c54e864e3c6";
+  // Init global variables for paging:
+
+  const pageLength = 10; // number of objects per page
+
+  let lon; // place longitude
+  let lat; // place latitude
+
+  let offset = 0; // offset from first object in the list
+  let count; // total objects count
```

Comment:
generally - do not want global variables, and should use state instead

also for globals -> make all caps i.e const TEMP = asdf